modules/rpc/thrift_test.go

```diff
@@ -27,18 +27,43 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"go.uber.org/fx/config"
 	"go.uber.org/fx/modules"
 	"go.uber.org/fx/service"
 	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/encoding/thrift"
 )
 
+type testHost struct {
+	service.Host
+	config config.Provider
+}
+
+func (h testHost) Config() config.Provider {
+	return h.config
+}
+
 func TestThriftModule_OK(t *testing.T) {
 	chip := ThriftModule(okCreate, modules.WithRoles("rescue"))
 	dale := ThriftModule(okCreate, modules.WithRoles("ranges"))
+	cfg := []byte(`modules:
+  rpc:
+    inbounds:
+     - tchannel:
+       port: 0
+     - http:
+       port: 0
```

Comment:
* It should be similar to inbounds, I am still wiring it together lots of different types, decided to make this first and add incrementally outbounds.
* Yes, right now there is no config format for the YARPC, it accepts a struct, that's why we need something very similar.